kit/blueprint/styles.scss

```diff
@@ -260,6 +260,22 @@ textarea.bp3-input,
   box-shadow: var(--xh-form-field-box-shadow);
 }
 
+body.xh-app {
+  .bp3-control,
+  .bp3-dark .bp3-control,
+  .bp3-dark .bp3-control.bp3-switch {
```

Comment:
This looks like it only targets `.bp3-switch` in dark theme? Is that what we want?

Reply:
I'm pretty sure it's targeting a bp3-control-indicator that is adjacent to a focused input,
when that bp3-control-indicator is contained by any of these:
`.bp3-control`   (control in light mode)
`.bp3-dark .bp3-control`  (control in dark mode)
`.bp3-dark .bp3-control.bp3-switch`.  (switch control in dark mode)

That's what I'm seeing happen.

Reply:
Yes... I guess my question was how come we're not targeting switch control in light mode?

Reply:
Oh, ok.  I've re-tested and find that .bp3-switch specificity is not needed now.  I've removed it.